Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.py

```diff
@@ -0,0 +1,729 @@
+                                             '?format=phantom' % context,
+                                  json_data=entities, timeout=30)
+
+
+def translate_score(score: int, threshold: int) -> int:
+    """Translate Recorded Future score to DBot score."""
+    if score >= threshold:
+        return Common.DBotScore.BAD
+    elif score >= 5:
+        return Common.DBotScore.SUSPICIOUS
+    else:
+        return Common.DBotScore.NONE
+
+
+def determine_hash(hash_value: str) -> str:
+    """Determine hash type by length."""
+    hash_length = len(hash_value)
+    if hash_length == 128:
+        return 'SHA512'
+    elif hash_length == 64:
+        return 'SHA256'
+    elif hash_length == 40:
+        return 'SHA1'
+    elif hash_length == 32:
+        return 'MD5'
+    elif hash_length == 8:
+        return 'CRC32'
+    else:
+        return 'CTPH'
+
+
+def level_to_criticality(level: int) -> str:
+    """Translate level integer to Criticality string."""
+    if level >= 4:
+        return 'Very Malicious'
+    elif level >= 3:
+        return 'Malicious'
+    elif level >= 2:
+        return 'Suspicious'
+    elif level >= 1:
+        return "Informational"
+    return 'Unknown'
+
+
+def rf_type_to_xsoar_type(entity_type: str) -> str:
+    if entity_type == 'IpAddress':
+        return 'ip'
+    elif entity_type == 'Hash':
+        return 'file'
+    elif entity_type == 'URL':
+        return 'url'
+    elif entity_type == 'CyberVulnerability':
+        return 'cve'
+    elif entity_type == 'InternetDomainName':
+        return 'domain'
+    raise DemistoException('Unknown Recorded Future '
+                           'entity type: %s' % entity_type)
+
+
+def prettify_time(time_string: str) -> str:
+    """Fix timestamps to a better format."""
+    if time_string:
+        parsed = datetime.strptime(time_string, "%Y-%m-%dT%H:%M:%S.%fZ")
+        return datetime.strftime(parsed, "%Y-%m-%d %H:%M:%S")
+    else:
+        return 'N/A'
+
+
+def create_indicator(entity: str, entity_type: str,
+                     score: int, description: str,
+                     location: Dict[str, Any] = {}) -> Common.Indicator:
+    """Create an Indicator object."""
+    thresholds = {'file': int(demisto.params().get('file_threshold', 65)),
+                  'ip': int(demisto.params().get('ip_threshold', 65)),
+                  'domain': int(demisto.params().get('domain_threshold', 65)),
+                  'url': int(demisto.params().get('url_threshold', 65)),
+                  'cve': int(demisto.params().get('cve_threshold', 65))}
+    dbot_score = translate_score(score, thresholds[entity_type])
+    dbot_description = 'Score above %s' % thresholds[entity_type] \
+        if dbot_score == Common.DBotScore.BAD else None
+    if entity_type == 'ip':
+        return Common.IP(entity,
+                         Common.DBotScore(entity, DBotScoreType.IP,
+                                          'Recorded Future', dbot_score,
```

Comment:
vendor name must be as integration name (I suggest adding it as a global constant)
```suggestion
                                          'Recorded Future v2', dbot_score,
```

Reply:
Should be fixed in the new commit.